libraries/key-identifier/src/key-identifier.js

```diff
@@ -39,15 +39,16 @@ export default class KeyIdentifier {
       ? derivationPath
       : DerivationPath.from(derivationPath)
 
+    Object.defineProperty(this, 'derivationPath', {
+      value: this.#derivationPath.toString(),
+      enumerable: true,
```

Comment:
```suggestion
      enumerable: true,
      writable: false,
```

Reply:
writeable defaults to false: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty#writable

(I also have a test for that which correctly fails)

Reply:
correctly fails in the node.js environment 😛 . But glad it defaults to false, resolving...